variables.tf

```
@@ -118,6 +118,12 @@ variable "create_pipes" {
   default     = true
 }
 
+variable "create_logging" {
```

`logging` is too vague
```suggestion
variable "create_log_delivery" {
```